dotcom-rendering/src/components/Discussion/Comments.stories.tsx

```diff
@@ -222,12 +213,8 @@ export const LoggedInHiddenNoPicks = () => {
 				comments={discussionMock.discussion.comments}
 				setComment={() => {}}
 				handleFilterChange={() => {}}
-				setTopFormActive={() => {}}
-				setReplyFormActive={setActive}
```

Comment:
With this refactoring we will loose interactivity from the stories but I think it's worth it. We've [said before](https://github.com/guardian/dotcom-rendering/pull/10311#discussion_r1465387059) we need to [add integration tests in discussion](https://github.com/guardian/dotcom-rendering/issues/10316).